pages/common/hub.md

```diff
@@ -11,6 +11,14 @@
 
```

Comment:
while you're at it, you could fix "another user repository" --> "another user's repository" as well :)